foods.html

```diff
@@ -1,20 +1,33 @@
 <!DOCTYPE html>
 <html>
 
+<a class="alert"> </a>
```

Comment:
This is a bit weird. Is this so you can add an error message in `all_food_objects`? you can just `.append()` to the body an html element